tests/playwright/src/model/workbench/status-bar.ts

```diff
@@ -37,4 +39,10 @@ export class StatusBar extends BasePage {
     await handleConfirmationDialog(this.page, 'Kind');
     await handleConfirmationDialog(this.page, 'Kind', true, 'OK');
   }
+
+  public async validateKubernetesContext(context: string): Promise<void> {
+    await playExpect(this.kubernetesContextButton).toBeVisible();
+    const currentContex = await this.kubernetesContextButton.textContent();
```

Comment:
it might be enough to just:
`await playExpect(kubernetesContext).toHaveText('context')`.